todo_app/templates/index.html

```diff
@@ -11,8 +11,19 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+        <li>{{ item.title }} -- {{ item.status }}</li>
+        <form id="completeItem{{item.id}}" method="POST" action="/complete/{{item.id}}">
```

Comment:
The form id is unnecessary.

Though it could be useful for automated tests in module 3, so fine to leave it in.

Reply:
ok